athlinks-schema.ttl

```diff
@@ -279,15 +225,16 @@ athlinks:subEventGroup
 
 athlinks:visibility
   a owl:DataTypeProperty ;
-  rdfs:comment "Denotes who has access to see an object"
+  rdfs:comment "Denotes who has access to see an object";
+  rdfs:label "Denotes who has access to see an object" ;
   rdfs:domain athlinks:AthleticEventGroup ;
   rdfs:range [ a xsd:string ;
                 owl:oneOf ("Private" "Friends" "Invite" "Public")
               ] .
 
 athlinks:recurrence
   a owl:DataTypeProperty ;
-  rdfs:comment "The default recurrence of a certain scheduling item"
+  rdfs:label "The default recurrence of a certain scheduling item";
```

Comment:
Changing comment to label is incorrect.
Please add a label without removing the comment.